TypewiseAlert/EmailAlert.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace TypewiseAlert
+{
+    public class EmailAlert : IAlert
+    {
+        public bool IsAlertPublished(Constants.BreachType breachType)
```

Comment:
Looks like return type "bool" introduced here to ease testability ...but in real world observers will not return any value for subjects

Reply:
Updated. please check